src/mode/modeVisual.ts

```diff
@@ -40,6 +40,9 @@ export class VisualMode extends Mode {
     }
 
     shouldBeActivated(key: string, currentMode: ModeName): boolean {
+        if (currentMode !== ModeName.Normal) {
+            return false;
+        }
         return key === "v";
```

Comment:
Whoa, been using vim for ages and never knew that `v` in visual mode brings you out of visual mode!